roaster-bom/pom.xml

```diff
@@ -155,6 +156,16 @@
 				<artifactId>jackson-datatype-jsr310</artifactId>
 				<version>${version.com.fasterxml.jackson.datatype.jackson-datatype-jsr310}</version>
 			</dependency>
+			<dependency>
+				<groupId>com.squareup.retrofit2</groupId>
+				<artifactId>retrofit</artifactId>
+				<version>${version.retrofit}</version>
+			</dependency>
+			<dependency>
+				<groupId>com.squareup.retrofit2</groupId>
+				<artifactId>converter-jackson</artifactId>
```

Comment:
nemtudom pontosan ez miert de van mar jackson es gson konverterunk is. Azok nem hasznosithatoak fel?